Simplified/NYPLBookDetailButtonsView.m

```diff
@@ -259,7 +259,7 @@ - (void)updateButtons
   
   BOOL fulfillmentIdRequired = NO;
   NYPLBookState state = [[NYPLBookRegistry sharedRegistry] stateForIdentifier:self.book.identifier];
-  BOOL hasRevokeLink = (self.book.revokeURL && state & (NYPLBookStateDownloadSuccessful | NYPLBookStateUsed));
+  BOOL hasRevokeLink = (self.book.revokeURL && state && (state == NYPLBookStateDownloadSuccessful || state == NYPLBookStateUsed));
```

Comment:
sorry I missed this before. You don't need the `&& state` because state is an integer